requirements/base.txt

```diff
@@ -371,6 +369,7 @@ werkzeug==2.3.3
     # via
     #   apache-superset
     #   flask
+    #   flask-appbuilder
```

Comment:
I think we need to add a CI linting rule to make sure the requirements files are always up to date (I assume this change didn't bring about this change). Any objections to me adding a step to the python lint CI workflow that does this in a separate PR?

Reply:
@villebro I agree. We do have [this](https://github.com/apache/superset/blob/master/.pre-commit-config.yaml#L59-L62) check, however this just makes sure that the SHAs are correct. Maybe we need to run `pip-compile-multi --no-upgrade` and make sure there's no resulting diff?

Reply:
> Maybe we need to run pip-compile-multi --no-upgrade and make sure there's no resulting diff?

@john-bodley yeah, I was thinking something along those lines